changelog/fix_doc_to_add_display_style_guide_option.md

```diff
@@ -0,0 +1 @@
+* Fix to add `-S`, `--display-style-guide`, and the option's description to the docs. ([@M-Yamashita01][])
```

Comment:
No change log entry is required because no user impact. Can you remove it?

Reply:
Thank you for pointing this out. We removed it.